Sources/ViewQuery.php

```diff
@@ -74,11 +74,11 @@ function ViewQuery()
 	<body id="help_popup">
 		<div class="tborder windowbg description">';
 
-	foreach ($_SESSION['debug'] as $q => $query_data)
+	foreach ($_SESSION['debug'] as $q => $qq)
```

Comment:
Nit: `$query_data` is a more descriptive name for a variable than `$qq`. Please change it back so that future developers will have an easier time reading your code. :)

Reply:
it's not my code,
this is the var name of logging.php